ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -417,6 +438,21 @@ FCU_DFSS_DFVSC_CHWRC_RHDHC:
   - CHWRC
   - RHDHC
 
+FCU_SFSS_STM_RTM:
+  description: "Variable speed DX fan coil with chilled water return temp control with drain pump monitoring ."
+  is_canonical: true
+  opt_uses:
+  - drain_pump_run_status
+  - supply_fan_speed_mode
+  - run_mode
+  - return_air_temperature_setpoint
+  implements:
+  - FCU
+  - SFSS
+  - STM
```

Comment:
FCUs only serve one space, directly, so this is DTM.

Reply:
Updated.